acme/acme/crypto_util.py

```diff
@@ -0,0 +1,146 @@
+"""Crypto utilities."""
+import contextlib
+import logging
+import socket
+import sys
+
+from six.moves import range  # pylint: disable=import-error,redefined-builtin
+
+import OpenSSL
+
+from acme import errors
+
+
+logger = logging.getLogger(__name__)
+
+
+_DEFAULT_SSL_METHOD = OpenSSL.SSL.SSLv23_METHOD
```

Comment:
After looking into this a bit, I think you're right @Kuba. While it probably isn't an issue, perhaps we could use `set_options` to exclude SSLv2 and SSLv3 protocols.